test/boost/cdc_test.cc

```diff
@@ -837,15 +838,18 @@ SEASTAR_THREAD_TEST_CASE(test_row_delete) {
         auto rows = dynamic_pointer_cast<cql_transport::messages::result_message::rows>(msg);
         BOOST_REQUIRE(rows);
         auto results = to_bytes(*rows);
-        BOOST_REQUIRE_EQUAL(results.size(), 3);  // 2 rows for update + 1 for delete
+        BOOST_REQUIRE_EQUAL(results.size(), 4);  // 1 insert + 2 updates + 1 row delete
```

Comment:
It's hard to read this test, not knowing which of the log rows happen because of which CQL commands. Could you have comments which part relates to which command?